### PR TITLE
top reviews widget time span #187

### PR DESCRIPTION
**Pull request by contactashish13**

### CODE REVIEW

File: includes/admin/models/class-wppr-query-model.php [hunk 1]
```diff
@@ -91,6 +91,7 @@ public function find(
 			'category_id'   => false,
 			'category_name' => false,
 			'post_type'     => array( 'post', 'page' ),
+			'post_date_range_weeks' => false,
```
[the remark this answers is not in the document]
Comment (contactashish13) [reply includes/admin/models/class-wppr-query-model.php]: @selul the filter does not really filter. The where sub-conditions are created by the $post array

[the remark this answers is not in the document]
Comment (selul) [reply includes/admin/models/class-wppr-query-model.php]: those are different, the filter does filter, returning products with min rating, price or which name contains a specific string. in the same fashion i would like to use the date.

[the remark this answers is not in the document]
Comment (contactashish13) [reply includes/admin/models/class-wppr-query-model.php]: please check https://github.com/Codeinwp/wp-product-review/blob/master/includes/admin/models/class-wppr-query-model.php#L218 - the actual where condition is built using $post not $filter. get_query_conditions builds the having clause only and I've followed the same template.

[the remark this answers is not in the document]
Comment (selul) [reply includes/admin/models/class-wppr-query-model.php]: @contactashish13 ah, fair enough

**Pull request merged**